0.17/Dockerfile

```diff
@@ -9,8 +9,8 @@ ARG PGID=845
 
 ENV PORT=34197 \
     RCON_PORT=27015 \
-    VERSION=0.17.36 \
-    SHA1=9ed258f4f27852e227ca5eac755c29874450f118 \
+    VERSION=0.17.37 \
+    SHA1=4af991cb42145854a4803203f31d42f81fea7034 \
```

Comment:
Can you please update with current master to hide this changes.

Reply:
ok done in #253 . 
This pull request can be cancelled.